server/user_config.go

```diff
@@ -100,6 +100,7 @@ type UserConfig struct {
 	SSLKeyFile             string          `mapstructure:"ssl-key-file"`
 	RestrictFileList       bool            `mapstructure:"restrict-file-list"`
 	TFDownloadURL          string          `mapstructure:"tf-download-url"`
+	TFDisableDownloads     bool            `mapstructure:"tf-disable-downloads"`
```

Comment:
This file controls the -- flag. Please add your flag to cmd/server.go too for the env var.

https://github.com/runatlantis/atlantis/blob/ce8c89b566d8a4d3cd70676af2deb6644a559802/cmd/server.go#L120

Reply:
Addressed in https://github.com/runatlantis/atlantis/pull/2843/commits/6ae09a25fd15c4e604fb4467f76dd91b10a9f91f